tests/src/reloadFunctionalTest/kotlin/org/jetbrains/compose/reload/tests/ErrorRecoveryTests.kt

```diff
@@ -126,4 +126,68 @@ class ErrorRecoveryTests {
             fixture.checkScreenshot("3-after-change")
         }
     }
+
+    @HotReloadTest
+    @QuickTest
+    fun `illegal code change - update compose entry function`(fixture: HotReloadTestFixture) = fixture.runTest {
```

Comment:
I think the test is great, maybe we should think of this whole concept of 'rejecting reloads' as a concept like 'validation'? Then could have a separate validation test and one test here that tests that we can actually recover from a broken validation?